src/rsa/public/key.rs

```diff
@@ -40,30 +39,11 @@ impl Key {
         // documentation for RSA keys.
 
         // Step 3 / Step c for `n` (out of order).
```

Comment:
This comment appears to have been moved into `Modulus::from_be_bytes`. Should it be removed here too?